src/applications/mhv/medications/api/rxApi.js

```diff
@@ -0,0 +1,38 @@
+import environment from 'platform/utilities/environment';
+import { apiRequest } from 'platform/utilities/api';
+import prescriptions from '../tests/fixtures/presciptions.json';
+
+const apiBasePath = `${environment.API_URL}/my_health/v1`;
+
+export const getPrescriptionList = () => {
+  return apiRequest(`${apiBasePath}/prescriptions?page=1&per_page=10`, {
+    headers: {
+      'Content-Type': 'application/json',
+    },
+  });
+};
+
+export const getPrescription = id => {
+  return apiRequest(`${apiBasePath}/prescriptions/${id}`, {
+    headers: {
+      'Content-Type': 'application/json',
+    },
+  });
+};
+
+export const mockGetPrescriptionList = () => {
```

Comment:
I'm not seeing where these mock functions are used? At any rate I'd suggest using the [existing mock API framework](https://github.com/department-of-veterans-affairs/vets-website#running-a-mock-api-for-local-development) so that the same code paths are used when testing and there aren't fixtures mixed in with implementation code. 🙂